### ISSUE TITLE
[MSFT-118] Challenges page: The topcoder logo doesn't have an alt text.

### ISSUE THREAD

**Issue by veshu** [importtest/ewe#80]
**Steps to Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 2. Ensure screen reader is switched on. 3. Press the shortcut 'ctrl + home' to make sure that you are at the top of the page. 4. Press the down arrow. 5. Notice screen reader reads 'link graphic www.topcoder'

**Expected Result**
Screen reader should announce the logo as 'link graphic topcoder logo'

**Actual Result**
Screen reader announces the topcoder logo incorrectly.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non-Text Content
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2029
